src/ECS/MessageTypes.h

```diff
@@ -13,4 +13,8 @@ PACK(struct health_s {
     int hp;
 });
 
+PACK(struct enemy_id_s {
+    long int value;
```

Comment:
long int ?

genre y auras plus de 2147483647 ennemy ?

Reply:
it will be unsigned int, is it ok ?, its same length of an int